src/Components/Setting/Setting.jsx

```diff
@@ -0,0 +1,35 @@
+import { React, useContext } from "react";
+import NameChanger from "../NameChanger/NameChanger";
+
+import './Setting.scss'
+const components = {
+  nameChanger: NameChanger,
+  // permission: PermissionManager,
+  // deleteRoomButton: deleteRoomButton,
+};
+
+function Setting({ settingType, title, info, props }) {
```

Comment:
```suggestion
function Setting({ settingType, title, info, ...props }) {
```
Komponente `prop` adında  property vermek yerine `settingType`, `title`, `info` dışında kalanları `props` içinde toplamak bence daha okunaklı ve kolay kullanımlıdır.

**Bu kullanıma geçildikten sonra ilgili komponetlerde de düzeltme yapılması gerekmektedir**